sdk/tests/extra/workload-simulator.html

```diff
@@ -76,192 +63,105 @@
 }
 </style>
 
-
-
 <canvas id=webglCanvas class=square></canvas>
 <canvas id=canvas class=square style=display:none;></canvas>
 <div id=dom class=square style=display:none;background:white>
   <img id=img style=width:256px;height:256px;background:black></img>
 </div>
-<div id="warning"></div>
+<div id=warning></div>
```

Comment:
I agree that it would have been a bad idea to use unquoted attributes when that article was written. However, I believe that unquoted attributes are safe to use now since HTML5.  To address the specific concerns raised in the article:

* HTML5 unquoted attributes can contain almost any character now including slashes, colons, and semicolons, so these attributes are all valid and the rules are much easier to follow.
* HTML5 parsing is standardized so browsers all agree and validators no longer accept the weird accidental syntax described on that page.
* There's no need for consistency with XML because there's no longer an expectation that HTML5 content will be transitioned to XML in the future.

Reply:
Got it. Thanks for correcting my understanding.